### ISSUE TITLE
just question

### ISSUE THREAD

**Issue by AingTii** [stuartpittaway/nanodesmapvmonitor#9]
hello,

@stuartpittaway 
sorry for contact this way...

Do you think its possible to use this with an esp32 with the inbuild bluetooth module?

I'm just programm a little as hobby, but not really well.
So maybe whta the importens Parts (.ino) to connect to the inverter an read the data?

I like to use an ESP32 to connect to the inverter and send the data via mqtt and wifi to iobroker.

Do you think its possible an base of your code?

Thank You
Jens

**Comment by stuartpittaway** [comment]
Yes I suspect it would work. I wrote this code a very long time ago, but have it working in several devices include raspberry pi and also using the python language
